dev/bender/plugins/ckeditor5/static/extensions.js

```diff
@@ -0,0 +1,90 @@
+/**
+ * @license Copyright (c) 2003-2015, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+/* globals bender, CKEDITOR, describe: true */
+
+'use strict';
+
+// ### General extensions.
+( function() {
+	// Make Bender wait to start running tests.
+	var done = bender.defer();
+
+	// Wait for the "ckeditor" module to be ready to start testing.
+	CKEDITOR.require( [ 'ckeditor' ], done );
+} )();
+
+// ### AMD related extensions.
+( function() {
+	var wasRequireCalled,
+		requires;
+
+	/**
+	 * AMD tools related to CKEditor.
+	 */
+	bender.amd = {
+		/**
+		 * Specifies the list of CKEditor modules to be loaded before tests start. The modules will be passed to the
+		 * describe() functions as parameters.
+		 *
+		 * @params {...String} module The name of the module to load.
+		 */
+		require: function() {
+			if ( wasRequireCalled ) {
+				throw 'bender.amd.require() must be called just once.';
+			}
+
+			wasRequireCalled = true;
+
+			var done = bender.defer();
+
+			var modules = [].slice.call( arguments );
+
+			CKEDITOR.require( modules, function() {
+				// Save all returned modules.
+				requires = [].slice.call( arguments );
+
+				// Call all describe()s that where waiting for `requires` to load.
+				flushDescribeQueue();
+
+				// Finally give green light for tests to start.
+				done();
+			} );
+		}
+	};
+
+	var originalDescribe = describe;
+	var describeQueue = [];
+
+	// Override the original Mocha's describe() so we can pass required modules to it.
+	describe = function( title, fn ) {
```

Comment:
I didn't give attention `describe.only()` and `describe.skip()` and I think it doesn't support them (it even undefines them). 

Making this work for `describe.skip()` is a piece of cake. As for `describe.only()` instead, we would have to do more work, but should be doable.

I think it is worth bringing support for it, even if we'll never use them, because there should be a transparent experience to coders trying to use all mocha's power.